velox/common/base/BloomFilter.h

```diff
@@ -96,6 +96,20 @@ class BloomFilter {
     }
   }
 
+  /// Tests a hashed input value directly on a serialized bloom filter.
+  /// For the implementation V1, this API involves no copy of the input
+  /// data.
+  static bool mayContain(const char* serializedBloom, uint64_t hashCode) {
+    common::InputByteStream stream(serializedBloom);
```

Comment:
If the serializedBloom is not a valid Velox serialized format, can it read the version and size correctly?

Reply:
The out-of-box behavior should be exactly the same with

```cpp
BloomFilter bloomfilter{};
bloomfilter.merge(data);
bool result = bloomfilter.mayContain(hashCode);
```

I think the deserialization safety is more or less guaranteed because we do check `kBloomFilterV1 == version` in both APIs.